frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -247,10 +247,11 @@ export const InventoryLayer: React.FC<InventoryLayerProps> = ({
   };
 
   // Fetch the geoJSON collection of properties.
+  const parcelDetail = !!selected?.parcelDetail ?? undefined;
```

Comment:
collate all falsey expressions to undefined.

Reply:
Should this not in a 'useMemo'